src/olympia/addons/tasks.py

```diff
@@ -720,7 +722,7 @@ def migrate_legacy_dictionary_to_webextension(addon):
     parsed_data = parse_addon(upload, user=user)
     # Create version.
     version = Version.from_upload(
-        upload, addon, platforms=[amo.PLATFORM_ALL.id],
+        upload, addon, selected_apps=[amo.FIREFOX.id],
```

Comment:
@diox does that work? Do you know if WebExtension dictionaries are compatible with Android?

Reply:
Yup